web/server/README.md

```diff
@@ -112,6 +112,15 @@ Example:
 
 For information how to configure the slaves to use TLS, check [securing the communication](../../streaming#securing-streaming-communications) in the streaming documentation. There you will find additional details on the expected behavior for client and server nodes, when their respective TLS options are enabled.
 
+When we define the use of SSL in a Netdata master, we are defining the behavior of the whole software, and case there is a conflict between the Netdata ports, the server will always assume the last rule given, for example, the next rules
```

Comment:
No, this isn't how it should work. See the comment on `socket.c`

Reply:
@joelhans I changed the documentation to explain the behavior of to define different SSL options for different connections, please, check whether it is necessary to clarify more for the users.